### PR TITLE
For vagrant-spk auto meteor, add kentonv:accounts-sandstorm

### PR DESCRIPTION
**Pull request by paulproteus**

### CODE REVIEW

**Comment by paulproteus** [comment]
Going to self-merge since this only affects an experimental codepath.

**Comment by paulproteus** [comment]
Comments welcome, though.

File: vagrant-spk [hunk 1]
```diff
@@ -271,6 +276,21 @@ def auto(args):
...
+    #
+    # - Maybe in the future, fix HTTP/HTTPS mixed content stuff on
+    #   common CDN domains. Note that the real fix for this is to
+    #   embed a copy of the assets in the app package, but this works
+    #   for now.
+    if stack == STACK_TYPE_METEOR:
+        ACCOUNTS_PKG = 'kenton:accounts-sandstorm'
+        if ACCOUNTS_PKG not in open('.meteor/packages').read():
+            with open('.meteor/packages', 'w') as fd:
+                fd.write(ACCOUNTS_PKG)
```
Comment (jparyani) [inline vagrant-spk]: I would add `fd.write('\n')` before this line to ensure the package starts on a newline.

File: vagrant-spk [hunk 2]
```diff
@@ -271,6 +276,21 @@ def auto(args):
...
+    # - Add the Sandstorm accounts package, for auto-sign-in.
+    #
+    # - Maybe in the future, fix HTTP/HTTPS mixed content stuff on
+    #   common CDN domains. Note that the real fix for this is to
+    #   embed a copy of the assets in the app package, but this works
+    #   for now.
+    if stack == STACK_TYPE_METEOR:
+        ACCOUNTS_PKG = 'kenton:accounts-sandstorm'
+        if ACCOUNTS_PKG not in open('.meteor/packages').read():
+            with open('.meteor/packages', 'w') as fd:
```
Comment (jparyani) [inline vagrant-spk]: This should be `open('.meteor/packages', 'a')`, not `'w'`

Comment (paulproteus) [inline vagrant-spk]: o snap​

**Pull request merged**